src/platform/silabs/BLEManagerImpl.h

```diff
@@ -25,15 +25,15 @@
 #pragma once
 #if CHIP_DEVICE_CONFIG_ENABLE_CHIPOBLE
 
-#ifdef RSI_BLE_ENABLE
+#if (BLE_ENABLE || RSI_BLE_ENABLE)
```

Comment:
Why two different defines for the same thing?

Reply:
We are using same `BleManagerImpl.h` for rs911x and siwx917 platforms. The BLE macros are not same in both SDKs.
Wiseconnect SDK which is used rs911x platforms uses `RSI_BLE_ENABLE`  and wifi sdk uses` BLE_ENABLE` macro. Based on these macros, BLE functions are enabled in SDKs.